packages/trace-viewer/src/sw/main.ts

```diff
@@ -54,6 +54,8 @@ async function loadTrace(traceUrl: string, traceFileName: string | null, clientI
     const backend = traceUrl.endsWith('json') ? new FetchTraceModelBackend(traceUrl) : new ZipTraceModelBackend(traceUrl, fetchProgress);
     await traceModel.load(backend, unzipProgress);
   } catch (error: any) {
+    if (error?.message === 'trace not found')
```

Comment:
Looks like you are mostly addressing the malformed (empty) JSON in live mode. Should we check that at parse time and return empty trac instead?

Reply:
That's a nice idea as well, but I think it'd be more complex. Just one example, we're treating empty traces as an error while parsing:

https://github.com/microsoft/playwright/blob/26a7dd4dd468bd15e0aa51dd5f190b7b72d8df4c/packages/trace-viewer/src/sw/traceModel.ts#L52-L53